src/App.jsx

```diff
@@ -51,19 +71,39 @@ function App() {
   }, [todoList, isLoading]);
 
   const addTodo = (newTodo) => {
-    setTodoList([...todoList, newTodo]);
+    setTodoList(prevTodoList => {
+      const updatedTodoList = [...prevTodoList, newTodo];
+      return updatedTodoList.sort((objectA, objectB) => {
```

Comment:
Given this is the same sorting code as above, think about abstracting the sort logic to a function to reduce repetition.